.kodiak.toml

```diff
@@ -4,12 +4,23 @@ version = 1
 [merge]
 
 method = "squash"
-
-notify_on_conflict = false
-
-dont_wait_on_status_checks = ["EIP Auto-Merge Bot", "Travis CI - Pull Request"]
-
+delete_branch_on_merge = true
 require_automerge_label = false
 
 # important
 block_on_neutral_required_check_runs = true
+
+# Add useful information & co-authors to the PR commit body
+[merge.message]
+include_coauthors = true
+include_pull_request_author = true
+include_pull_request_url = true
+
+# Keep dependencies up-to-date automatically 
+[merge.automerge_dependencies]
+versions = ["minor", "patch"] # do not automerge major version upgrades
+usernames = ["dependabot"]
+
+# Automatically keep PRs ahead of the main branch
+[update]
+always = true
```

Comment:
Bots shouldn't edit commits on other people's PRs.

Reply:
Hm, what does `update` mean in this context? I assumed it meant rebase on top of the default branch.

Reply:
That's exactly it, but it fundamentally doesn't edit the authors' commits.

Reply:
That is literally fundamentally editing the authors' commits. The hash will change, meaning they'll need to force pull. That's extremely impolite.

Reply:
Fair. I'll remove it.